ckan/logic/action/get.py

```diff
@@ -1536,7 +1536,7 @@ def package_autocomplete(context, data_dict):
             'name:{0}',
             'title:{0}',
         ]).format(search.query.solr_literal(q)),
-        'fl': ['name', 'title'],
```

Comment:
fl doesn't accept lists anymore?

Reply:
It seems, that SOLRv3 doesn't accept lists, but all newer versions are ok with them